### PR TITLE
Add buildpy

### PR DESCRIPTION
**Pull request by kshramt**

### CODE REVIEW

**Comment by grst** [comment]
@conda-forge/staged-recipes LGTM

File: recipes/buildpy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+source:
+  url: https://files.pythonhosted.org/packages/a9/4d/c4a1a46a34fcd285d15d93d32f8daaf5c0f28a72adde156897dbd527a3b0/buildpy-6.2.4.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv"
+  skip: True
```
Comment (scopatz) [inline recipes/buildpy/meta.yaml]: This doesn't actually build anything.  You should remove this line.

Comment (kshramt) [reply recipes/buildpy/meta.yaml]: Thank you for your comment. I removed `skip: True`.

Comment (kshramt) [reply recipes/buildpy/meta.yaml]: I replaced the line with `skip: True  # [win or py2k]`.

**Pull request merged**